axonserver/src/main/java/io/axoniq/axonserver/localstorage/file/Synchronizer.java

```diff
@@ -152,22 +165,26 @@ public synchronized void init(WritePosition writePosition) {
         }
     }
 
-    public void forceCurrent() {
-        if (updated.compareAndSet(true, false)) {
-            WritePosition writePosition = currentRef.get();
-            if (writePosition != null) {
-                writePosition.force();
-            }
+    private void forceCurrent() {
+        WritePosition writePosition = updated.getAndSet(null);
+        if (writePosition != null) {
+            lastForced.set(writePosition.sequence);
+            forceTimer.record(writePosition::force);
```

Comment:
Thank you for including this one! I guess that's valuable to see if the underlying storage is slow (or we are too fast).